src/lambdas/get-location.ts

```diff
@@ -0,0 +1,46 @@
+import fetch from 'node-fetch'
+
+/*
+Return the coordinates of a given address using the MapBox Places API
+
+eg. provide an address to query = '1600 Pennsylvania Ave NW, Washington, DC'
+returns {
+  name: 'Central Park, New York, New York, United States',
+  coordinates: [ -76.982011, 38.879239 ],
+}
+
+Environment Variables:
+  * MAPBOX_API_BASE_URL = 'https://api.mapbox.com/geocoding/v5/mapbox.places' (default)
+  * MAPBOX_API_TOKEN    = 'pk.token-goes-here'
+*/
+
+export const handler = async (event: any = {}): Promise<any> => {
+  let queryAddress = event.arguments.address
+  if (!queryAddress) {
+    return { statusCode: 400, body: `Error: You are missing the address argument` }
```

Comment:
the `statusCode` property is not necessary as you are using AppSync and GraphQL. Errors should be mapped using the responseMappingTemplate

Reply:
You're right! This is not the ApiGateway :). Removed status codes and updated the response mapping templates for both lambdas.